hieradata/class/vagrant/logs_elasticsearch.yaml

```diff
@@ -0,0 +1,20 @@
+---
+
+govuk_elasticsearch::aws_access_key: 'ABCDEFGHIJKLMNOPQRST'
+govuk_elasticsearch::aws_secret_key: 'n4AQffsnBJ5rcQK9DBn5eYHVuDqhT5DVCYa7ZU9tt'
+
+lv:
+  data:
+    pv: '/dev/sdb1'
+    vg: 'elasticsearch'
+
+mount:
+  /mnt/elasticsearch:
+    disk: '/dev/mapper/elasticsearch-data'
+    govuk_lvm: 'data'
+    mountoptions: 'defaults'
```

Comment:
I don't think we need to set either the `lv` or the `mount` properties in this file - they'll fall back to the production config which should work.